addons/pandora/backend/entity_backend.gd

```diff
@@ -354,6 +356,65 @@ func save_data() -> Dictionary:
 		"_properties": _serialize_data(_properties)
 	}
 
+func import_data(data: Dictionary) -> int:
```

Comment:
usually, I try to agree on one type of "error handling flow", e.g.:

- return status code
- throw exception (not quite possible in Godot)
- use dedicated `*_failure` signals

For Pandora you will see I try to use failure signals as much as possible. So instead of returning any data, introduce another signal called `import_failure` that gets emitted when something goes wrong.